frontend/public/components/utils/units.js

```diff
@@ -134,6 +133,14 @@ const round = units.round = (value, options) => {
   return Intl.NumberFormat(locales, rest).format(value);
 };
 
+const round = units.round = (value, fractionDigits) => {
+  if (!isFinite(value)) {
+    return 0;
```

Comment:
Again, not sure treating non finite values as 0 is correct

Reply:
This is the behavior which we had before tests started to break.
Humanize functions return object which is supposed to look like (with added types)
```
value: number;
unit: string;
string: string;
```
`round` function is operating on `value` part and result should be number.